enterprise/coderd/schedule/template.go

```diff
@@ -350,14 +350,23 @@ func (s *EnterpriseTemplateScheduleStore) updateWorkspaceBuild(ctx context.Conte
 		return nil
 	}
 
+	// Calculate the new autostop max_deadline from the workspace. Since
+	// autostop is always calculated from the build completion time, we don't
+	// want to use the returned autostop.Deadline property as it will likely be
+	// in the distant past.
+	//
+	// The only exception is if the newly calculated workspace TTL is is now
+	// zero, which means the workspace can now stay on indefinitely.
```

Comment:
typo:
```suggestion
	// The only exception is if the newly calculated workspace TTL is now
	// zero, which means the workspace can stay on indefinitely.
```

Reply:
I would say "now" is correct here, but I'll fix the "is is"